Makefile

```diff
@@ -2,3 +2,11 @@ SUB_DIRS = utils/fabric-ccenv-sgx ercc ecc_enclave ecc tlcc_enclave tlcc
 
 all clean test:
 	$(foreach DIR, $(SUB_DIRS), $(MAKE) -C $(DIR) $@;)
+
+checks: linter
+
+linter:
+	@echo "LINT: Running code checks.."
+	@./scripts/golinter.sh
```

Comment:
Given that we realistically require fabric source already checked out, couldn't we just reference the scripts from the fabric distro. Or did you have to change the scripts so you have to include them here?   If not, we can just reference them via ../../hyperledger/fabric.

Apropos fabric dependencies: Unless we make FABRIC_PATH environment variable mandatory, we should also change tlcc_enclave/generate_protos.sh correspondingly along the lines of below patch

```
diff --git a/tlcc_enclave/generate_protos.sh b/tlcc_enclave/generate_protos.sh
index 3d6e40f..7de9fe9 100755
--- a/tlcc_enclave/generate_protos.sh
+++ b/tlcc_enclave/generate_protos.sh
@@ -2,7 +2,7 @@

 # set -eux

-FABRIC_PATH=${FABRIC_PATH-~/fabric}
+FABRIC_PATH=${FABRIC_PATH-../../../hyperledger/fabric}
 NANOPB_PATH=${NANOPB_PATH-~/nanopb}

 PROTOC_OPTS="--plugin=protoc-gen-nanopb=$NANOPB_PATH/generator/protoc-gen-nanopb"
```

make checks currently fails for me with the following. Is this expected? (I guess that's what #62 supposedly fixes?)
```
> LINT: Running code checks..
> Checking with gofmt
> Checking with goimports
> Checking for golang.org/x/net/context
> Checking with go vet
> # github.com/hyperledger-labs/fabric-private-chaincode/ecc
> ecc/enclave_chaincode_test.go:223:13: result of fmt.Errorf call not used
> ecc/enclave_chaincode_test.go:228:13: result of fmt.Errorf call not used
> ecc/enclave_chaincode_test.go:235:13: result of fmt.Errorf call not used
> Makefile:9: recipe for target 'linter' failed
```

Lastly, goimports is not part of go tools, so we probably should also mention in README that you have to do a .'go get golang.org/x/tools/cmd/goimports'?

Reply:
1. Yes, we could indeed try to reuse `golinter.sh` from Fabric and call it directly. However, if we maintain or own we can easily change it according to our needs. I don't have a strong opinion here. This just works :)

2. yes, code check fails at the moment. #62 fixes that. Looking forward to the c/cpp code checks.

3. I added it to the linter script.